rosdep/base.yaml

```
@@ -2814,6 +2814,13 @@ libgphoto-dev:
     precise: [libgphoto2-2-dev]
     raring: [libgphoto2-2-dev]
     saucy: [libgphoto2-6-dev]
+libgpiod-dev:
+  arch: [libgpiod]
+  debian: [libgpiod-dev]
+  fedora: [libgpiod-devel]
+  opensuse: [libgpiod]
```

The other rules provide development files, this one should too (I checked the spec, and this subpackage is provided).

```suggestion
  opensuse: [libgpiod-devel]
```